doc/release-notes.md

```diff
@@ -274,6 +277,12 @@ To enable block pruning set `prune=<N>` on the command line or in
 `bitcoin.conf`, where `N` is the number of MiB to allot for
 raw block & undo data.
 
+A value of 0 disables pruning. The minimal value above 0 is 550. Your
+wallet is as secure with high values as it is with low ones. Higher
+values merely reduce the network traffic in case of reorganization of
```

Comment:
I think this is a confusing sentence.  Choosing a larger value means that a node can recover from larger reorgs; however the minimum value of 550MiB is designed to allow for reorgs up to 288 blocks (~2 days).

If there happened to be a reorg that required more data on disk than the user had (very unlikely) then the user's node will crash, and the user would have to restart the node with -reindex.  In the case of a node that was pruned, running -reindex will cause the node to delete all the data on disk, which can't (currently) be used by bitcoind when doing a -reindex.  So in particular the user will end up redownloading the whole blockchain in such a scenario (the amount re-downloaded will not depend on the value passed to -prune -- unless somehow the user selects a value large enough that all the block files remained on disk and no pruning actually took place).

Rather than try to re-explain pruning here, maybe it would make sense to refer readers back to the 0.11.0 release notes?  I don't know how easy that is to link to I guess.